hyperliquid/websocket_manager.py

```diff
@@ -30,6 +30,8 @@ def subscription_to_identifier(subscription: Subscription) -> str:
         return f'userFundings:{subscription["user"].lower()}'
     elif subscription["type"] == "userNonFundingLedgerUpdates":
         return f'userNonFundingLedgerUpdates:{subscription["user"].lower()}'
+    elif subscription["type"] == "webData2":
+        return "webData2"
```

Comment:
You should include user in the identifier so that multiple webData2 subscriptions can be multiplexed

Reply:
fixed